AMW_angular/io/src/app/deployments/deployments.component.ts

```diff
@@ -289,22 +296,22 @@ export class DeploymentsComponent implements OnInit {
     );
   }
 
-  copyURL() {
+  showToast: boolean = false;
+
+  async copyURL() {
     const url: string = decodeURIComponent(window.location.href);
-    $('body')
-      .append(
-        $(
-          '<input type="text" name="fname" class="textToCopyInput" style="opacity:0"/>'
-        ).val(url)
-      )
-      .find('.textToCopyInput')
-      .select();
     try {
-      document.execCommand('copy');
-    } catch (e) {
+      await navigator.clipboard.writeText(url);
+      console.log('URL copied to clipboard');
```

Comment:
With the new toast and the window prompt if something fails, there is no need to log to the console.